qiskit_experiments/library/randomized_benchmarking/clifford_utils.py

```diff
@@ -216,6 +199,119 @@ def _unpack_num_multi_sigs(self, num, sigs):
             for k in sig:
                 sig_size *= k
             if num < sig_size:
-                return [i] + self._unpack_num(num, sig)
+                return [i] + cls._unpack_num(num, sig)
             num -= sig_size
         return None
+
+    @classmethod
+    def transpile_single_clifford(cls, cliff_circ: QuantumCircuit, basis_gates: List[str]):
+        """Transpile a single clifford circuit using basis_gates."""
+        backend = AerSimulator()
+        return transpile(cliff_circ, backend, optimization_level=1, basis_gates=basis_gates)
+
+    @classmethod
+    def num_from_clifford_single_gate(cls, inst, qubits, rb_num_qubits, basis_gates):
+        """
+        This method does the reverse of clifford_1_qubit_circuit and clifford_2_qubit_circuit -
+        given a clifford, it returns the corresponding integer, with the mapping
+        defined in the above method.
+        The mapping is in the context of the basis_gates. Therefore, we define here
+        the possible supersets of basis gates, and verify that the given inst belong to
+        one of these sets.
+        """
+        name = inst.name
+        gates_with_delay = basis_gates.copy()
+        gates_with_delay.append("delay")
+        single_gate_map = (
```

Comment:
A slightly more elegant way of doing this:
```
cliff_single_gate_maps = {1: CLIFF_SINGLE_GATE_MAP_1Q, 2: CLIFF_SINGLE_GATE_MAP_2Q}
single_gate_map = cliff_single_gate_maps[rb_num_qubits]
```

Reply:
Answering to the comment above regarding @staticmethod: this method actually calls another class method: `cls.clifford_1_qubit_circuit(num)` so I think it must be a classmethod. Or am I missing something? I did change a couple of other methods to static, as you suggested.

Reply:
Regarding your second comment - I agree. Nicer!